draft-ietf-gnap-core-protocol.md

```diff
@@ -2368,108 +2370,199 @@ unknown_request
+- prompt the RO for consent for the requested delegation
+- describe to the RO what information is being released, to whom, and for what purpose
+- provide warnings to the RO about potential attacks or negative effects of allowing the information
+- allow the RO to modify the client instance's requested access, including limiting or expanding that access
+- provide the RO with artifacts such as receipts to facilitate an audit trail of authorizations
+- allow the RO to deny the requested delegation
+
+The AS is also allowed to request authorization from more than one RO, if the AS deems fit. For example, a medical
+record might need to be released by both an attending nurse and a physician, or both owners of a bank account
+need to sign off on a transfer request. Alternatively, the AS could require N of M possible RO's
+to approve a given request in order. The AS could also determine that the end user is not the appropriate RO
+for a given request and reach out to the appropriate RO asynchronously. The details of determining which RO's are required for a given
+request are out of scope for this specification.
+
+The client instance can also indicate that it is capable of facilitating interaction with the end user,
+another party, or another piece of software through its [interaction start](#request-interact-start) request.
+In many cases, the end user is delegating their own access as RO to the client instance. Here, the 
+AS needs to determine the identity of the end user and will often need to interact directly with
+the end user to determine their status as an RO and collect their consent. If the AS has determined
+that authorization is required and the AS can support one or more of the requested interaction start
+methods, the AS returns the associated [interaction start responses](#response-interact). The client
+instance SHOULD [initiate one or more of these interaction methods](#interaction-start) in order to 
+facilitate the granting of the request. If more than one interaction start method is available,
+the means by which the client chooses which methods to follow is out of scope of this specification.
+The client instance MUST use each interaction method once at most.
+
+After starting interaction, the client instance can then make a [continuation request](#continue-request)
+either in response to a signal indicating the [finish of the interaction](#interaction-finish), through
+polling, or through some other method defined by an extension of this specification.
+
+If the AS and client instance have not reached a state where the delegation can be granted, the
+AS and client instance can repeat the interaction process as long as the AS supplies the client
+instance with [continuation information](#response-continue) to facilitate the ongoing requests.
+
+## Interaction Start Methods {#interaction-start}
+
+To initiate an interaction start method indicated by the
```

Comment:
We should probably shortly remind here what we mean by "interaction start". Notice that the link to 3.3 never talks about start (so far it's only discussed in the request, but it's only implicit in the response)